src/Client.php

```diff
@@ -259,7 +260,29 @@ public function getNotification( $notificationId ){
         $path = sprintf( self::PATH_NOTIFICATION_LOOKUP, $notificationId );
 
         return $this->httpGet( $path );
+    }
+
+    /**
+     * This returns the pdf contents of a letter notification.
+     *
+     * @param string $notificationId
+     *
+     * @return string
+     */
+    public function getPdfForLetter( $notificationId ){
+
+        $path = sprintf( self::PATH_NOTIFICATION_PDF, $notificationId );
 
+        $url = $this->createURL( $path, [] );
+        $response = $this->makeGetRequest( $url );
+
+        switch( $response->getStatusCode() ){
+            case 200:
+                // true passed to getBody gets the raw string data
```

Comment:
I don't fully understand this comment, in particular the `true` bit?

Reply:
Also am I correct in thinking that we don't do a simple `$this->httpGet` here because the return is a stream and we need to call `getBody()->getContents()` on it to get a string for the user?

Reply:
ah the true comment is no longer applicable, i'll squash that out.

but yes that's correct, `$this->httpGet` tries to parse a json response so isn't suitable for this

Reply:
Maybe mention why we can't use httpGet in your new comment please :)